lib/cb/version.rb

```diff
@@ -1,3 +1,3 @@
 module Cb
-  VERSION = '16.2.1'
+  VERSION = '16.2.2'
```

Comment:
@julman I'm tempted to say this is a major version bump to v17.0.0 instead of a patch bump. Reason being: does this count as a change to the public interface? Technically you're still just passing in a big ol args hash, but one of those expected keys in the hash changed. Then again... that expected key was wrong anyway, so it might as well of not existed previously.... so then technically it is backward compatible now right? JESUS I'VE TALKED MYSELF IN A CIRCLE